content/100-getting-started/02-setup-prisma/100-start-from-scratch/110-relational-databases/150-using-prisma-migrate.mdx

```diff
@@ -537,25 +537,25 @@ In this guide, you'll use [Prisma Migrate](/concepts/components/prisma-migrate)
 
 ```prisma file=prisma/schema.prisma copy
 model Post {
-  id        Int      @id @default(autoincrement())
+  id        BigInt   @id @default(autoincrement())
   createdAt DateTime @default(now())
   updatedAt DateTime @updatedAt
   title     String   @db.VarChar(255)
   content   String?
   published Boolean  @default(false)
   author    User     @relation(fields: [authorId], references: [id])
-  authorId  Int
+  authorId  BigInt
```

Comment:
Note: I've updated these relation scalars to `BigInt` too to match what comes back from Introspection in the [Add to existing database](https://www.prisma.io/docs/getting-started/setup-prisma/add-to-existing-project/relational-databases/introspection-typescript-cockroachdb) guide

Reply:
Did you confirm that the demo SQL results in BigInts?

```
CREATE TABLE "public"."User" (
  id SERIAL PRIMARY KEY NOT NULL,
  name VARCHAR(255),
  email VARCHAR(255) UNIQUE NOT NULL
);
```

Reply:
This is valid but prisma migrate will never generate migrations with `SERIAL` on CockroachDB.

The cockroach equivalent would be

```
CREATE TABLE "User" (
  id INT8 PRIMARY KEY NOT NULL DEFAULT unique_rowid(),
  name STRING(255),
  email STRING(255) UNIQUE NOT NULL
);
```

Reply:
I'll take the time to do that.

Reply:
I cockroachified the SQL a first time, but that involved changing types, and that would cascade into rewriting the rest of the schemas in the page, I'll give it another shot trying to keep the introspection output to what is expected here.

Ideally, that would be tested with running code, I can't guarantee things will match exactly without spending even more time reproducing the workflow and comparing outputs.